recipes/asdf-unit-schemas/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "asdf-unit-schemas" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asdf_unit_schemas-{{ version }}.tar.gz
+  sha256: 42b78d67213efe4ffd4529fb0e58d9c7a0dab5cbf8839b230f1bc0a446bff999
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - importlib-resources >=3  # [py<39]
```

```suggestion
    - importlib-resources >=3
```

This should not be necessary